[Security Solution] add warning message for duplicate blocklist values

joeypoon opened the pull request:
## Summary

Add warning messages for blocklist duplicate values
![Mar-28-2022 22-23-12](https://user-images.githubusercontent.com/11009772/160527428-ddee753f-7e10-4016-a62e-0dd27ac6094d.gif)

Ref: https://github.com/elastic/kibana/issues/128445

elasticmachine commented:
Pinging @elastic/security-onboarding-and-lifecycle-mgt (Team:Onboarding and Lifecycle Mgt)

joeypoon commented:
Ref: https://github.com/elastic/kibana/pull/128708

ashokaditya reviewed (commented):
Nice 🔥 
Works as expected with the warning message.

I notice that the `Add blocklist` button gets enabled when we change any value on the form (the form is rerendered). Is that expected? It should be enabled only when all required fields (name and a condition) are added and are valid. Right?

Here're a a couple of clips.
![blocklist-clip](https://user-images.githubusercontent.com/1849116/160614636-4160ae71-aa71-4adb-b096-431596a8daa8.gif)
![blocklist-clip-2](https://user-images.githubusercontent.com/1849116/160615579-70f540bc-46e3-4a42-ba2d-755e9c725d95.gif)

joeypoon commented:
thanks for catching that @ashokaditya. I had updated the interface for `ItemValidation` but neglected to update the `isValid` logic to account for it. fixed now.

ashokaditya reviewed (approved):
🚢  it!

dasansol92 reviewed (approved):
Left a question but other than that it looks great!

joeypoon commented on x-pack/plugins/security_solution/public/management/pages/blocklist/translations.ts:
```diff
@@ -113,13 +113,25 @@ export const ERRORS = {
+  DUPLICATE_VALUES: i18n.translate(
+    'xpack.securitySolution.blocklists.warnings.values.duplicateValues',
+    {
+      defaultMessage: 'One or more duplicate values removed',
```
No, will double check with her. For `This value exists`, that only happens when you're typing a single value so the value itself is right there.
